### ISSUE TITLE
Idea - Show documentation from docblock.

### ISSUE THREAD

**Issue by hotoiledgoblinsack** [Peekmo/atom-autocomplete-php#34]
Hello

This is more of an idea, but since docblocks are already parsed, it would be neat if the description (short and/or long summary) from the docblock could be shown when selecting an item in the autocompletion list. I believe atom-ternjs [1] also does this for JavaScript methods. Optionally the docblocks of the parameters could also be shown in the description (e.g. by making the parameter names bold and placing the description behind them in the popup).

For methods without a docblock, the docblock from the parent class or interface should probably be selected (but I think the same already applies for the current code).

Thanks in advance

[1] https://github.com/tststs/atom-ternjs

**Comment by Peekmo** [comment]
Hello,

In fact, it's that hard and I would like to do it since the functionality appears in autocomplete + plugin. But I prefered till now, to focus on bigger autocompletion functionalities (because I'm using this plugin at work). 

I'll see, I can't say when (perhaps very soon because you asked it ?), but I really want to have the doc too.

**Comment by hotoiledgoblinsack** [comment]
Thanks for your response. Feel free to take your time with this, it is not something I desperately need but more of a suggestion on the longer term (or farther along the roadmap).

**Comment by Peekmo** [comment]
I think that I'll try on it after #28 
But if you want, you can also try to implement this functionality ;) Every contribution is welcome !

**Comment by Peekmo** [comment]
Hello

The short description is now shown (no inheritance, just the short description on the given method of the given class for the moment)

**Comment by hotoiledgoblinsack** [comment]
Thanks, it's very handy already even without the inheritance.

**Comment by hotoiledgoblinsack** [comment]
Can probably be closed as well, you already had the summary working and the (long) description is now shown in the tooltip, with support for inheriting it from parent classes.

**Comment by Peekmo** [comment]
I will add tomorrow, a command to show the tooltip under the cursor. I'll close this issue once it's done.

**Comment by Peekmo** [comment]
If someone know how to get an element under the cursor, it would be great (see https://discuss.atom.io/t/get-dom-element-under-cursor/20840).

**Comment by hotoiledgoblinsack** [comment]
Can you perhaps use `pixelPositionForBufferPosition` with the cursor position and then elementFromPoint on the shadowRoot with the detected position? Note also that if the cursor is outside the view, you probably can't get the element as the only elements in the HTML are the ones that are currently visible, I think.

**Comment by hotoiledgoblinsack** [comment]
Also, perhaps it's a good idea to split the remaining work off into a separate ticket, to have a better ticket title and get rid of the comments that no longer apply.

**Comment by hotoiledgoblinsack** [comment]
I've split this off into #182 so this can be closed (the title is confusing ;-).
